resources/osx-au/Info.plist

```diff
@@ -32,7 +32,7 @@
 	<key>CFBundleIconFile</key>
 	<string>surgeicon</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.vemberaudio.audiounit.surge</string>
+	<string>com.vemberaudio.plugins.surge</string>
```

Comment:
I don't understand? if you were developing this in Xcode you would have this as a versioned file. What's not good exactly?

Reply:
@baconpaul Sorry, wasn't very clear -- i changed the AU and VST bundle identifiers to the same value in order to look up the bundle in code to copy the resources. It seems like that might be a bad thing if both VST and AU were loaded in the same host with the same bundle identifier, no? I am really clueless about this aspect of osx libraries etc.

Reply:
... the idea is maybe if this was `$BUNDLE_IDENTIFIER` or something, and then also `#define`d in code, and referenced [here](https://github.com/kurasu/surge/pull/92/files#diff-699a99990780e396a819e6ed03008700R38) by the define instead of hardcoding this, the identifiers could be different. I just don't really know what the right thing to do is.